connectors/migrations/20230906_3_github_fill_parents_field.ts

```diff
@@ -0,0 +1,101 @@
+import {
+  getDiscussionDocumentId,
+  getIssueDocumentId,
+} from "@connectors/connectors/github/temporal/activities";
+import { updateDocumentParentsField } from "@connectors/lib/data_sources";
+import {
+  Connector,
+  GithubDiscussion,
+  GithubIssue,
+} from "@connectors/lib/models";
+
+async function main() {
+  // if first arg is "all", update all connectors, else update only the
+  // connector for the corresponding workspace id
+  const connectors =
+    process.argv[2] === "all"
+      ? await Connector.findAll({
+          where: {
+            type: "github",
+          },
+        })
+      : await Connector.findAll({
+          where: {
+            type: "github",
+            workspaceId: process.argv[2],
```

Comment:
I guess it doesn't matter much since it a one off, but curious what happens if I forget argv[2] ? Does it basically does the same as `"all"` because `workspaceId: undefined` doesn't filter ?

Reply:
Nice catch. Let's validate that